### PR TITLE
More server stuff

### PR DESCRIPTION
**Pull request by NachoToast**
- testing
- use dotenv

### CODE REVIEW

File: server/src/handlers/getNumLobbies.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        expect(data0).toBe(0);
+
+        serverHub['_lobbies']['fakeLobbyA'] = {} as Lobby;
+
+        const { data: data1 } = await httpClient.get('/numLobbies');
+        expect(data1).toBe(1);
+
+        serverHub['_lobbies']['fakeLobbyB'] = {} as Lobby;
+        serverHub['_lobbies']['fakeLobbyC'] = {} as Lobby;
+        serverHub['_lobbies']['fakeLobbyD'] = {} as Lobby;
```
Comment (tukib) [inline server/src/handlers/getNumLobbies.test.ts]: love the private field access ❤️ ❤️ ❤️ ❤️

**Review by tukib — APPROVED** [review]
pog

File: server/src/handlers/getNumLobbies.test.ts (hunk 1 above)
Comment (NachoToast) [reply server/src/handlers/getNumLobbies.test.ts]: yeah actually pog

Comment (PolarWolf314) [reply server/src/handlers/getNumLobbies.test.ts]: based

**Comment by PolarWolf314** [comment]
this is so based

**Pull request merged**